x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/Ccr.java

```diff
@@ -132,11 +132,6 @@
 public class Ccr extends Plugin implements ActionPlugin, PersistentTaskPlugin, EnginePlugin, RepositoryPlugin, ClusterPlugin {
 
     public static final String CCR_THREAD_POOL_NAME = "ccr";
-    public static final String CCR_CUSTOM_METADATA_KEY = "ccr";
```

Comment:
Perhaps leave a comment here, stating that these moved to CcrConstants in 7.x (since this targets 7.x only)?

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/73237/commits/b80e0e99d1f83f10afc0cbce0f85e724d92dd9b0